scriptworker/exceptions.py

```diff
@@ -55,3 +55,10 @@ class ScriptWorkerTaskException(ScriptWorkerException):
     def __init__(self, *args, exit_code=1, **kwargs):
         self.exit_code = exit_code
         super(ScriptWorkerTaskException, self).__init__(*args, **kwargs)
+
+
+class DownloadError(ScriptWorkerTaskException):
+    def __init__(self, msg):
+        super(DownloadError, self).__init__(
+            msg, exit_code=3
```

Comment:
I think 'malformed-payload' means the task definition was wrong, while a DownloadError might mean there's an infrastructure problem.  I'm not sure if we should use a 4, 5, or 1; I've been guessing so far.  Luckily we can adjust these pretty easily.